worker-script.js

```diff
@@ -201,7 +201,7 @@ const getHomePageContents = () => {
       }
 
       const response = await fetch(
-        "https://radio-france-rss.aerion.workers.dev/search/?query=" +
+        "${baseUrl}/search/?query=" +
```

Comment:
I thought about it later, but if one wants to host it, it's probably better to have baseUrl as an environment variable (as it's settable in both cloudflare worker & locally).
I'll keep it in mind if needing to debug later :)